core/src/main/java/org/jruby/RubyBignum.java

```diff
@@ -236,6 +236,54 @@ public static BigInteger long2big(long arg) {
      *  ================
      */
 
+    /** rb_big_digits
+     *
+     */
+    @Override
+    public RubyArray digits(ThreadContext context, IRubyObject base) {
+        BigInteger self = (this).getValue();
+        if (self.compareTo(new BigInteger("0")) == -1) {
+            throw context.getRuntime().newMathDomainError("out of domain");
```

Comment:
Throw context.runtime into a local variable at the top and reuse it, in the style of other methods.